### DISCUSSION TITLE
Is it possible to use a private docker container on Akash?

### DISCUSSION THREAD

**Discussion by 621625** [akash-network/community#395] [General]
Hi, 

I'm Mr. Climber, a very friendly guy.

Is it possible to use a private docker container on Akash? I don't feel like uploading my docker containers to docker hub.

**Comment by anilmurty** [comment]
Not currently - but it is on our roadmap to work on supporting secrets and private container registries. by the way - it may be easier to ask technical questions on our discord server

**Comment by gavinharriss** [comment]
It is now possible to use private container registries - https://akash.network/docs/getting-started/stack-definition-language/#private-container-registry-support
